feature-group-definitions/border-image.yml

```diff
@@ -1,4 +1,5 @@
 name: Border images
+description: The `border-image` CSS property draws an image around an element's border.
```

Comment:
Hmm, doesn't it replace the painting of the border, and not painting around it?

Reply:
How about this?

```suggestion
description: The `border-image` CSS property draws an image around an element.
```